.github/workflows/cut-release-branch.yaml

```diff
@@ -32,11 +32,15 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8 # pin@v3
+        with:
+          token: ${{ secrets.CUT_RELEASE_BRANCH_CREDENTIALS }}
```

Comment:
does the APTOS_BOT_GH_PAT not have these creds?

Also is this an aptos bot PAT or deploy key? (if PAT, nit can we prefix the secret name with APTOS_BOT or something so we know where this cred comes from)

Reply:
It might, I figured it would be better to have a token per function